package.json

```diff
@@ -43,7 +43,7 @@
     ]
   },
   "scripts": {
-    "docker:build": "docker build . -t braveheartsoftware/static-dicomweb:latest",
+    "docker:build": "docker build . -t braveheartsoftware/static-dicomweb:arm",
```

Comment:
Please don't change this to arm - you should just be able to build it as "latest" and then run that one locally.